prow/cmd/config-bootstrapper/BUILD.bazel

```diff
@@ -36,3 +37,11 @@ go_binary(
     name = "config-bootstrapper",
     embed = [":go_default_library"],
 )
+
+prow_image(
+    name = "image",
+    base = "@git-base//image",
+    directory = "/",
+    files = [":config-bootstrapper"],
```

Comment:
I don't think the `directory` or `files` lines are necessary?
https://github.com/kubernetes/test-infra/blob/37917141d969c178688714727b7f942f1bf2287e/prow/cmd/hook/BUILD.bazel#L6-L10

Reply:
addressed